### DISCUSSION TITLE
pages/about/

### DISCUSSION THREAD

**Discussion by giscus** [MHuiG/talk#2] [Giscus]
# pages/about/

热爱技术，热爱创造。正在探秘事物本质与联系的探索者。喜欢遥望星星存在过的地方。 在Alone的海洋里有一只孤独的鲸，没有放弃海洋的它终会在最后找到同频率的伙伴。 --52赫兹的鲸

https://blog.mhuig.top/pages/about/

**Comment by MHuiG** [comment]
welcome!

**Comment by rantrism** [comment]
您好～我是腾讯云+社区的运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。  
我们诚挚的邀请您并期待您的加入~

**Comment by wndbac** [comment]


**Reply by MHuiG** [reply]
哪个按钮？

**Reply by MHuiG** [reply]
```
{% btn center solid large, 博客大事记, /pages/timeline/, fad fa-paper-plane %}
```

**Reply by MHuiG** [reply]
https://volantis.js.org/v6/tag-plugins/#button

**Comment by wndbac** [comment]


**Reply by MHuiG** [reply]
```yml
sidebar:
  for_page: [webinfo]
```

**Comment by duchengyao** [comment]
萌ICP备 哈哈哈哈哈

**Reply by MHuiG** [reply]
嗯，你没看错，这是萌国的ICP备案。

**Reply by duchengyao** [reply]
所以 https://mhuig.top/privacy-policy/ 和 首页那个公钥是什么意思呀，有什么隐藏的组织么

**Reply by MHuiG** [reply]
这也是组织的阴谋吗？这一切都是命运石之门的选择！

**Reply by duchengyao** [reply]
虽然不太懂，但加油^_^

**Comment by StarWEB890** [comment]
博主，能否把Travelling的链接更换成 https://travellings.cn/ ？
（https://travellings.link/ 部分地区访问不了，我这边访问出现BUG，详情：https://github.com/travellings-link/travellings/discussions/1214 ）

**Reply by MHuiG** [reply]
我反代了一下 https://blog.mhuig.top/travellings/

顺便把 `urls.splice(url.indexOf(origin)` 替换为 `urls.splice(urls.indexOf(origin)`

另外 [travellings.link](https://travellings.link/assets/travelling.js) 目前有 627 组数据，[travellings.cn](https://travellings.cn/assets/travelling.js) 目前只有 612 组数据

**Comment by bucketchen** [comment]
作者你好，我在使用submit-urls-from-sitemap-to-search-engine仓库模版的时候在github actions里看见所有工作都已被跳过，这是为什么

**Reply by MHuiG** [reply]
https://github.com/MHG-LAB/submit-urls-from-sitemap-to-search-engine/blob/main/.github/workflows/push.yml#L12

**Reply by bucketchen** [reply]
是要更改为自己的仓库地址吗，仓库的token是要在secrets的actions里创建是嘛Qwq

**Reply by bucketchen** [reply]
已经解决啦，谢谢Qwq

**Reply by bucketchen** [reply]
大佬，为什么推送百度前50条可以，到百度随机就不行了，配额是够的，错误代码：Traceback (most recent call last):
[7](https://github.com/bucketchen/submit-urls-from-sitemap-to-search-engine/actions/runs/3328190780/jobs/5503899355#step:7:8)
  File "generate.py", line 44, in <module>
[8](https://github.com/bucketchen/submit-urls-from-sitemap-to-search-engine/actions/runs/3328190780/jobs/5503899355#step:7:9)
    googleUrllist=googleUrllist + random.sample(result,50)
[9](https://github.com/bucketchen/submit-urls-from-sitemap-to-search-engine/actions/runs/3328190780/jobs/5503899355#step:7:10)
  File "/opt/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/random.py", line 363, in sample
[10](https://github.com/bucketchen/submit-urls-from-sitemap-to-search-engine/actions/runs/3328190780/jobs/5503899355#step:7:11)
    raise ValueError("Sample larger than population or is negative")
[11](https://github.com/bucketchen/submit-urls-from-sitemap-to-search-engine/actions/runs/3328190780/jobs/5503899355#step:7:12)
ValueError: Sample larger than population or is negative
[12](https://github.com/bucketchen/submit-urls-from-sitemap-to-search-engine/actions/runs/3328190780/jobs/5503899355#step:7:13)
Error: Process completed with exit code 1.

**Reply by MHuiG** [reply]
可能是你没那么多文章，这边默认有100+， random.sample(result,50) 把50 改小一点，实在不行就改成0

**Reply by bucketchen** [reply]
好咧

**Comment by talefou** [comment]
大佬 想问一下您的顶部栏的阴影是如何去除的呢
![](https://bu.dusays.com/2024/06/27/667d59c3f372b.png)
我在配置中去除了```[shadow]``` 仅保留了```[]``` 但是如下图所示
![](https://bu.dusays.com/2024/06/27/667d5a16d4e0a.png)
